packages/cli/src/deploy/resolveConfig.ts

```diff
@@ -44,8 +44,10 @@ export function resolveConfig<config extends ConfigInput>({
     .map(toFunctionSignature);
 
   const systems = Object.entries(resolvedConfig.systems).map(([systemName, system]): System => {
-    const namespace = config.namespace;
-    const name = system.name;
+    const resource = labelToResource(systemName);
```

Comment:
why are we parsing this when `system` already has `system.namespace` and `system.name`?

Reply:
This `System` type doesn't have a separate namespace and name? https://github.com/latticexyz/mud/blob/main/packages/world/ts/config/resolveWorldConfig.ts#L81

Reply:
ahh that's an oversight and we should fix that!

Reply:
Parsing the resource in `resolveSystemConfig` now!